### PR TITLE
Question Section Code Review

### PR DESCRIPTION
**Pull request by olaassem**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@ <h2 class="screen-reader">The problem</h2>
           </blockquote>
           <cite class="problem__author">
             World Health Organization, 2005
-            <a href="http://www.who.int/" target="_blank"><sup class="problem__sup">3</sup></a>
+            <sup class="problem__sup"><a class="a-sup" href="http://www.who.int/" target="_blank">3</a></sup>
```
Comment (mzlock) [inline public/index.html:66]: ah, cool 👍 

I'll resolve my comment on the earlier PR

File: public/index.html [hunk 2]
```diff
@@ -96,29 +96,29 @@ <h3 class="keyFactors__list__hdr">Cost</h3>
...
           <h3 class="keyFactors__list__hdr">Complexity</h3>
           <p class="keyFactors__list__content">“Taking 15 medications every day is tough. Especially since they all have different instructions. How am I supposed to keep track of everything?”</p>
-        </div>
+        </div> -->
       </div>
     </div>
   </section>
 
-        <!-- <section id="question-section" class="question">
+<section id="question-section" class="question">
```
Comment (mzlock) [inline public/index.html:104]: indentation!

File: src/stylesheets/base/_typography.scss [hunk 3]
```diff
@@ -29,6 +29,18 @@
     letter-spacing: none;
   }
 }
+
+.a-sup {
+  @include text($primary-font, normal, 1rem, $color-orange-red);
```
Comment (mzlock) [inline src/stylesheets/base/_typography.scss:35]: A couple of questions/notes here:
- do you need all of this CSS here? This mixin is adding 4 lines of CSS
- does this need to be a class? `sup a {}` seems pretty specific
- does the `sup` still have any default styles that will/can help you here?
- is this link style any different from your other in-body-copy link styles?

File: src/stylesheets/base/_typography.scss [hunk 4]
```diff
@@ -29,6 +29,18 @@
...
+
+  &:hover {
+    font-weight: bold;
+  }
```
Comment (mzlock) [inline src/stylesheets/base/_typography.scss:39]: changing the font-weight on hover is generally a bad idea. It makes your link take up more room than pre-hover and can give you some funny text flashing/wrapping/moving

File: src/stylesheets/base/_typography.scss [hunk 5]
```diff
@@ -29,6 +29,18 @@
...
+  @media (min-width: 550px) {
+    font-size: 1rem;
+  }
```
Comment (mzlock) [inline src/stylesheets/base/_typography.scss:43]: this hasn't changed from the previous breakpoint. Need this?

File: src/stylesheets/base/_typography.scss [hunk 6]
```diff
@@ -99,11 +117,36 @@
...
       }
     }
   }
 }
+
+
+//QUESTION
+.question {
+  // color: $color-dark-blue;
+  // text-align: center;
```
Comment (mzlock) [inline src/stylesheets/base/_typography.scss:148]: still need these? delete?

File: src/stylesheets/layouts/_factors.scss [hunk 7]
```diff
@@ -1,12 +1,11 @@
...
-  text-align: center;
 
   &__wrapper {
     position: relative;
     @include flex;
-    width: 100%;
     max-width: 77rem;
+    width: 100%;
     margin: 0;
     padding: 2.5rem 1.5rem;
```
Comment (mzlock) [inline src/stylesheets/layouts/_factors.scss:10]: 🔤

File: src/stylesheets/layouts/_factors.scss [hunk 8]
```diff
@@ -69,8 +57,8 @@
 
         @media (min-width: 550px) {
           max-width: 23rem;
-          padding: 1rem 1rem 1rem 2.5rem;
-          transform: translate(70%, 100%);
+          padding: 1rem;
+          transform: translate(70%, -220%);
           overflow-x: visible;
```
Comment (mzlock) [inline src/stylesheets/layouts/_factors.scss:61]: 🔤

File: src/stylesheets/layouts/_problem.scss [hunk 9]
```diff
@@ -23,11 +26,11 @@
   }
 
   &__intro {
-    @include problem-marginbtm;
+    margin-bottom: $content-margin-btm;
   }
 
   &__context {
-    @include problem-marginbtm;
+    margin-bottom: $content-margin-btm;
```
Comment (mzlock) [inline src/stylesheets/layouts/_problem.scss:31]: I like this as a variable better 👍

File: src/stylesheets/layouts/_question.scss [hunk 10]
```diff
@@ -0,0 +1,80 @@
...
+  margin-top: 1.5rem;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0rem;
+    padding: 2.5rem 1.5rem;
```
Comment (mzlock) [inline src/stylesheets/layouts/_question.scss:14]: 🔤

Comment (mzlock) [reply src/stylesheets/layouts/_question.scss:14]: Also - same note - watch how much you're using flexbox

File: src/stylesheets/layouts/_question.scss [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+  &__context {
+    margin-bottom: $content-margin-btm;
+
+    @media (min-width: 550px) {
+      margin-right: 20rem;
+    }
+
+    @media (min-width: 550px) {
+      margin-right: 10rem;
+    }
```
Comment (mzlock) [inline src/stylesheets/layouts/_question.scss:45]: you have two `min-width: 550px` breakpoints here

Comment (mzlock) [reply src/stylesheets/layouts/_question.scss:45]: I'm curious how this `margin-right` comes into play

File: src/stylesheets/layouts/_question.scss [hunk 12]
```diff
@@ -0,0 +1,80 @@
...
+    @media (min-width: 800px) {
+      margin-bottom: 2.6rem;
+    }
+  }
+
+
+
```
Comment (mzlock) [inline src/stylesheets/layouts/_question.scss:52]: extra space!

File: src/stylesheets/layouts/_question.scss (hunk 11 above)
Comment (mzlock) [reply src/stylesheets/layouts/_question.scss:45]: is the margin leaving space for the phone?

File: src/stylesheets/layouts/_question.scss [hunk 13]
```diff
@@ -0,0 +1,80 @@
...
+    @media (min-width: 550px){
+      margin-right: 0;
+    }
+  }
+
+  &__phone {
+    position: absolute;
+    right: 0px;
+    width: 130px;
+    transform: rotate(6deg) translate(0%, -140%);
```
Comment (mzlock) [inline src/stylesheets/layouts/_question.scss:65]: cooool 👍

File: src/stylesheets/layouts/_question.scss [hunk 14]
```diff
@@ -0,0 +1,80 @@
...
+
+    @media (min-width: 400px) {
+      width: 150px;
+      transform: rotate(6deg) translate(0%, -160%);
+    }
+    // @media (min-width: 550px) {
+    //   // width: 220px;
+    //   // transform: rotate(6deg) translate(8%, 25%);
+    // }
```
Comment (mzlock) [inline src/stylesheets/layouts/_question.scss:74]: still need this?

**Review by mzlock — CHANGES_REQUESTED** [review]
More changes!

File: src/stylesheets/base/_typography.scss (hunk 4 above)
Comment (olaassem) [reply src/stylesheets/base/_typography.scss:39]: ok. resolved in PR #4

File: src/stylesheets/base/_typography.scss (hunk 6 above)
Comment (olaassem) [reply src/stylesheets/base/_typography.scss:148]: resolved in PR #4

File: src/stylesheets/layouts/_factors.scss (hunk 7 above)
Comment (olaassem) [reply src/stylesheets/layouts/_factors.scss:10]: resolved in PR #4

File: src/stylesheets/layouts/_factors.scss (hunk 8 above)
Comment (olaassem) [reply src/stylesheets/layouts/_factors.scss:61]: Resolved in pr #4

File: src/stylesheets/base/_typography.scss (hunk 3 above)
Comment (olaassem) [reply src/stylesheets/base/_typography.scss:35]: - removed the class.
- deleted mixin.
- yes, the sup link style is different from others.
resolved  in pr#4

File: src/stylesheets/base/_typography.scss (hunk 5 above)
Comment (olaassem) [reply src/stylesheets/base/_typography.scss:43]: deleted. resolved in pr#4

File: src/stylesheets/layouts/_question.scss (hunk 10 above)
Comment (olaassem) [reply src/stylesheets/layouts/_question.scss:14]: resolved in pr#4

File: src/stylesheets/layouts/_question.scss (hunk 11 above)
Comment (olaassem) [reply src/stylesheets/layouts/_question.scss:45]: - deleted second margin. typo!!! resolved in pr#4
- yes, the space is for the phone. i use the margins and i have noticed that you use padding. is there a reason you use padding?

File: src/stylesheets/layouts/_question.scss (hunk 12 above)
Comment (olaassem) [reply src/stylesheets/layouts/_question.scss:52]: resolved in pr#4

File: src/stylesheets/layouts/_question.scss (hunk 14 above)
Comment (olaassem) [reply src/stylesheets/layouts/_question.scss:74]: nope. resolved in pr#4

**Pull request merged**